DevelopedCalculator.js

```diff
@@ -1,6 +1,7 @@
 class DevelopedCalculator extends Calculator {
 
     render() {
+        super.history = null;
```

Comment:
why `history` is defined on the `super` not on the `DevelopedCalculator`?
you must have reason for that?

Reply:
I did not understand your first question!

Reply:
`super` refers to the parent class. in this case the super/parent class is `Calculator` class.
my question why you're defining the `history` property on the parent class not `DevelopedCalculator`

I assume you have a reason for doing this?